src/python/WMComponent/JobAccountant/AccountantWorker.py

```diff
@@ -472,6 +472,18 @@ def handleJob(self, jobID, fwkJobReport):
             elif jobType == "Merge" and set(outputMap) == {'MergedRAW', 'MergedErrorRAW', 'logArchive'} \
                     and outputModules == {'MergedErrorRAW', 'logArchive'}:
                 pass
+            elif jobType == "Merge" and set(outputMap) == {'MergedL1SCOUT', 'MergedErrorL1SCOUT', 'logArchive'} \
+                    and outputModules == {'MergedL1SCOUT', 'logArchive'}:
+                pass
+            elif jobType == "Merge" and set(outputMap) == {'MergedL1SCOUT', 'MergedErrorL1SCOUT', 'logArchive'} \
+                    and outputModules == {'MergedErrorL1SCOUT', 'logArchive'}:
+                pass
+            elif jobType == "Merge" and set(outputMap) == {'MergedHLTSCOUT', 'MergedErrorHLTSCOUT', 'logArchive'} \
+                    and outputModules == {'MergedHLTSCOUT', 'logArchive'}:
+                pass
+            elif jobType == "Merge" and set(outputMap) == {'MergedHLTSCOUT', 'MergedErrorHLTSCOUT', 'logArchive'} \
+                    and outputModules == {'MergedErrorHLTSCOUT', 'logArchive'}:
+                pass
```

Comment:
This is a quick-and-dirty solution that works. A nicer fix would be to use `fwjrFile['dataset'].get('dataTier', '')` to avoid repeating the code, defining:
```
mergedModule="Merged"+dataset
mergedErrorModule="MergedError"+dataset
```

```suggestion
            elif jobType == "Merge" and set(outputMap) == {mergedModule, mergedErrorModule, 'logArchive'} and 'logArchive' in outputModules and (mergedModule in outputModules or mergedErrorModule in outputModules)
                pass
```

Nonetheless, I'm not sure if that would brake other workflows in which this verification is necessary

Reply:
what do you think @amaltaro

Reply:
I have the following suggestion to replace all those checks for merge jobs:
```
elif jobType == "Merge" and (set(outputMap) & {'MergedErrorRAW', 'MergedErrorL1SCOUT', 'MergedErrorHLTSCOUT'}):
  pass
```

In other words, if it's a merge that does not match `set(outputMap) == outputModules` but it has one of the expected Error output modules, just let it through.

Reply:
I like your suggestion as is it a lot cleaner. The part I don't like is that it still means that, in the future, if we want to add a new data tier, we will still need to find this rather obscure code section.

I guess we can see this as a feature, not a bug, as it prevents typos in data tier names, but I would prefer to have explicit data tier validation in the repack Workload factory, rather than here (#11954).

Do you see a problem in using `mergedErrorModule="MergedError"+datatier`?

Reply:
Hi German, I think that would work as well. It would add a few extra lines of code to do the parsing, add new output module and run the final comparison, but still it would work.